sfofferchrome/payload.js

```diff
@@ -1,13 +1,8 @@
 // send the page title as a chrome message
 
-var s = document.createElement("script");
-s.src="https://code.jquery.com/jquery-3.4.1.js";
-s.onload = function(e){ /* now that its loaded, do something */ }; 
-document.head.appendChild(s); 
-$(document).ready(function(){
-
 //SF Account:
-var account = $("table.detailList th:contains('Account')").siblings("td").first().text()
+var account = $("#summary-val").text()

```

Comment:
how is var account renamed to  (#summary-val") ? Please explain this syntax revisory. Thanks!

Reply:
This change should be removed before merging. I don't have access to Saleforce and so I was testing using Jira which has an element with the id "summary-val".